handlers/handler_api.go

```diff
@@ -0,0 +1,93 @@
+package handlers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// RegisterSQSHandler registers endpoints
+func (handler *SQSHandler) RegisterSQSHandler() {
+	http.HandleFunc("/server", handler.ServiceHandler)
+	http.HandleFunc("/sqs", handler.SQSHandler)
+	http.HandleFunc("/job", handler.HandleJob)
+}
+
+// ServiceHandler handles stop/start the SQS querrying service
+func (handler *SQSHandler) ServiceHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "PUT" {
+		val := r.URL.Query().Get("start")
+		if val == "true" {
+			handler.StartServer()
+		} else if val == "false" {
+			handler.ShutdownServer()
+		}
```

Comment:
this could return just a small JSON of success for shutdown/startup, or error if not `true` or `false`

Reply:
Have more detail in 0bdaf510f2e3325148cb8f863171cbae8b1f5855